parsons/__init__.py

```diff
@@ -93,7 +92,9 @@
     ("parsons.zoom.zoom", "Zoom"),
 ):
     try:
-        globals()[connector_name] = getattr(importlib.import_module(module_path), connector_name)
+        globals()[connector_name] = getattr(
```

Comment:
I think this change here breaks the linting. potentially relevant is that a recent PR upgraded us from black 22 to black 24, which has some differences in how it lints files